navigation.js

```diff
@@ -209,22 +192,6 @@ define('navigation',
                 el.getAttribute('rel') === 'external';
     }
 
-    z.body.on('click', 'a', function(e) {
-        var href = this.getAttribute('href');
-        var $elm = $(this);
-        var preserveScrollData = $elm.data('preserveScroll');
-        // Handle both data-preserve-scroll and data-preserve-scroll="true"
-        var preserveScroll = typeof preserveScrollData !== 'undefined' && preserveScrollData !== false;
```

Comment:
This used to ensure we didn't scroll to the top of the screen  e.g. the old tabs navigation used it otherise you'd scroll up and miss the new tab content that got loaded in. If we're not using it any more then we can bin it but not sure if that use-case might come up in the future? 

I noticed that when we do logins for apps e.g. paid apps we lose the scroll position in the parent due to the reload. That's potentially a similar case.